src/screens/AuthScreen/Util.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+export const getUserData = async (url: string) => {
+  if (url === 'https://www.realdevsquad.com/goto') {
+    return axios
+      .get(`https://api.realdevsquad.com/users/self`, {
```

Comment:
we can assign variables to apis and use them respectively, the benefit of that would be changing those variable apis at one place would reflect across the app and become easier to work with!